Fix VERSION.
Fix mandir.
Use gnutls instead of openssl.
Fix overlinking.
Add -L flag for MacPorts libraries.
--- librtmp/Makefile.orig	2021-02-19 11:54:54.000000000 -0600
+++ librtmp/Makefile	2022-04-22 22:41:19.000000000 -0500
@@ -1,11 +1,11 @@
-VERSION=v2.4
+VERSION=2.4
 
 prefix=/usr/local
 
 incdir=$(prefix)/include/librtmp
 bindir=$(prefix)/bin
 libdir=$(prefix)/lib
-mandir=$(prefix)/man
+mandir=$(prefix)/share/man
 BINDIR=$(DESTDIR)$(bindir)
 INCDIR=$(DESTDIR)$(incdir)
 LIBDIR=$(DESTDIR)$(libdir)
@@ -16,27 +16,26 @@
 AR=$(CROSS_COMPILE)ar
 
 SYS=posix
-CRYPTO=OPENSSL
-#CRYPTO=GNUTLS
+#CRYPTO=OPENSSL
+CRYPTO=GNUTLS
 DEF_POLARSSL=-DUSE_POLARSSL
 DEF_OPENSSL=-DUSE_OPENSSL
 DEF_GNUTLS=-DUSE_GNUTLS
 DEF_=-DNO_CRYPTO
-REQ_GNUTLS=gnutls,hogweed,nettle
+REQ_GNUTLS=gnutls
 REQ_OPENSSL=libssl,libcrypto
-PUB_GNUTLS=-lgmp
 LIBZ=-lz
 LIBS_posix=
 LIBS_darwin=
 LIBS_mingw=-lws2_32 -lwinmm -lgdi32
-LIB_GNUTLS=-lgnutls -lhogweed -lnettle -lgmp $(LIBZ)
-LIB_OPENSSL=-lssl -lcrypto $(LIBZ)
-LIB_POLARSSL=-lpolarssl $(LIBZ)
+LIB_GNUTLS=-L$(prefix)/lib -lgnutls $(LIBZ)
+LIB_OPENSSL=-L$(prefix)/lib -lssl -lcrypto $(LIBZ)
+LIB_POLARSSL=-L$(prefix)/lib -lpolarssl $(LIBZ)
 PRIVATE_LIBS=$(LIBS_$(SYS))
 CRYPTO_LIB=$(LIB_$(CRYPTO)) $(PRIVATE_LIBS)
 CRYPTO_REQ=$(REQ_$(CRYPTO))
 CRYPTO_DEF=$(DEF_$(CRYPTO))
-PUBLIC_LIBS=$(PUB_$(CRYPTO))
+PUBLIC_LIBS=$(LIBZ) $(PUB_$(CRYPTO))
 
 SO_VERSION=1
 SOX_posix=so
